### PR TITLE
zsh-you-should-use: init at 1.1.0

### PR DESCRIPTION
**Pull request by Ma27**
###### Motivation for this change

Simple zsh plugin that reminds you to use existing aliases. With
`oh-my-zsh` it can be installed like this:

``` nix
{
  programs.zsh.ohMyZsh = {
    customPkgs = [ pkgs.zsh-you-should-use ];
    plugins = [ "you-should-use" ];
  };
}
```

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/shells/zsh/zsh-you-should-use/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "MichaelAquilina";
+    repo = pname;
+    rev = version;
+    sha256 = "0fig5ralagi5jajk7gdm52jvwql17qk9cd6j98qsndvckb26a753";
+  };
+
+  dontBuild = true;
```
Comment (mmahut) [inline pkgs/shells/zsh/zsh-you-should-use/default.nix]: Does using ``zunit`` makes any sense for us?

File: pkgs/shells/zsh/zsh-you-should-use/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  installPhase = ''
+    install -D you-should-use.plugin.zsh $out/share/zsh/plugins/you-should-use/you-should-use.plugin.zsh
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/MichaelAquilina/zsh-you-should-use;
```
Comment (mmahut) [inline pkgs/shells/zsh/zsh-you-should-use/default.nix]: Minor cosmetic issue: can you please quote this?

**Review by mmahut — APPROVED** [review]
- [X] reviewed the diff and commit messages
- [X] made sure ofBorg build succeeded for all applicable platforms
- [X] run nix-review without any failures
- [X] run and tested the output

File: pkgs/shells/zsh/zsh-you-should-use/default.nix (hunk 2 above)
Comment (Ma27) [reply pkgs/shells/zsh/zsh-you-should-use/default.nix]: Have we reached some consensus which syntax should be preferred? The majority of all packages still uses this syntax.

File: pkgs/shells/zsh/zsh-you-should-use/default.nix (hunk 1 above)
Comment (Ma27) [reply pkgs/shells/zsh/zsh-you-should-use/default.nix]: (on a side-note: `dontBuild` isn't used to disable tests).

But do you think it is? It doesn't seem too widely adopted :)

Comment (mmahut) [reply pkgs/shells/zsh/zsh-you-should-use/default.nix]: Sorry, my brainfart :) I read ``dontCheck`` and was checking the zunit tests in there.

File: pkgs/shells/zsh/zsh-you-should-use/default.nix (hunk 2 above)
Comment (mmahut) [reply pkgs/shells/zsh/zsh-you-should-use/default.nix]: Nothing official yet, it's up to you.

**Comment by Ma27** [comment]
As this is a fairly trivial package: unless there are any objections, I'd merge this tomorrow.

File: pkgs/shells/zsh/zsh-you-should-use/default.nix (hunk 2 above)
Comment (Ma27) [reply pkgs/shells/zsh/zsh-you-should-use/default.nix]: Yeah, I'd stick to the current convention for now. Regardless of what comes out in the end, this kind of change can be automated.

**Pull request merged**